js/ui/layout.js

```diff
@@ -309,6 +309,11 @@ const LayoutManager = new Lang.Class({
         this.emit('hot-corners-changed');
     },
 
+    setViewsClone: function(actor) {
+        this._viewsClone = actor;
```

Comment:
I'll do, thanks for pointing it out. Yet another case of code brought from the old shell